setup/src/installer.ts

```diff
@@ -288,12 +296,29 @@ async function ghcupGHCHead(): Promise<void> {
     'install',
     'ghc',
     '-u',
-    'https://gitlab.haskell.org/ghc/ghc/-/jobs/artifacts/master/raw/ghc-x86_64-deb9-linux-integer-simple.tar.xz?job=validate-x86_64-linux-deb9-integer-simple',
+    'https://gitlab.haskell.org/ghc/ghc/-/jobs/artifacts/master/raw/ghc-x86_64-deb11-linux-integer-simple.tar.xz?job=validate-x86_64-linux-deb9-integer-simple',
```

Comment:
no idea if deb11 is good, hoping that CI will show

Reply:
Really, this URL is a bit of a hack and I'd love it if ghcup supported 'head' natively, but I understand that it's a tricky ask. (It would require essentially having a "nightly" build available somewhere official rather than grabbing things from gitlab directly)